src/Uno.UI/UI/Xaml/Media/Animation/Timeline.animation.cs

```diff
@@ -0,0 +1,473 @@
+﻿using Uno.Extensions;
+using System;
+using System.Collections.Generic;
+using Uno.Disposables;
+using System.Text;
+using System.Linq;
+using Uno.Diagnostics.Eventing;
+using Windows.UI.Core;
+using Uno.Logging;
+using Uno.UI.DataBinding;
+
+namespace Windows.UI.Xaml.Media.Animation
+{
+	public partial class Timeline
+	{
+		private protected partial class AnimationImplementation<T> : IDisposable where T : struct
+		{
+			private readonly static IEventProvider _trace = Tracing.Get(TraceProvider.Id);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [A static field in a generic type is not shared among instances of different close constructed types.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5301553)